includes/gateway/class-omise-payment.php

```diff
@@ -132,6 +132,26 @@ public function protectMetadata($protected, $metadataKeys)
 		return $protected;
 	}
 
+	/**
+	 * get pending status
+	 * 
+	 * This function is crate to get value for pending status,
+	 * since we cannot mock constant values for unit test.
+	 */
+	public function get_pending_status() {
+		return self::STATUS_PENDING;
+	}
+
+	/**
+	 * file_get_contents wrapper
+	 * 
+	 * Since we cannot mock global function,
+	 * we have to create a wrapper for file_get_contents.
+	 */
+	public function file_get_contents( $url ) {
+		return file_get_contents( $url );
+	}
```

Comment:
Can we move these to their own helper functions?

Reply:
I have moved `file_get_contents` to its own helper class. 
For `get_pending_status`, I think it still can be under  `Omise_Payment` since it is to get it's own constant.